integration/spark/app/src/test/java/io/openlineage/spark/agent/ArgumentParserTest.java

```diff
@@ -104,7 +105,7 @@ void testConfToHttpConfig() {
   void testDeprecatedConfig() {
     SparkConf sparkConf =
         new SparkConf()
-            .set("spark.openlineage.host", URL)
+            .set("spark.openlineage.url", URL)
```

Comment:
What will happen if a user doesn't provide `host` nor `url`? Will we throw some user friendly exception with a message, or a NullPointerException as observed previously?